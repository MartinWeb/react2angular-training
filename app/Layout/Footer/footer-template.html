<footer>
    Chuck Norris Â© {{$ctrl.date | date:"d/MM/yyyy"}}
</footer>