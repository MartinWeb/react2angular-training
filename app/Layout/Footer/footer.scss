footer {
    position: fixed;
    bottom: 0;
    left: 0;
    width: 100%;
    padding: 10px 40px;
    background-color: rgb(241, 90, 36);
    color: white;
}