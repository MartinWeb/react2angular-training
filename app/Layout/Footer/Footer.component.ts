import angular from "angular";
import template from './footer-template.html';
import './footer.scss';

export class FooterController implements angular.IController {
    public date: Date;

    public $onInit() {
        this.date = new Date(1993, 3, 21);
    }
}

export const FooterComponent: angular.IComponentOptions = {
    template,
    controller: FooterController
};