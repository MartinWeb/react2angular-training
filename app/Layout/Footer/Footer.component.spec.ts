import { FooterComponent } from "./Footer.component";

describe('footerComponent', () => {
    test('should have valid snapshot', () => {
        expect(FooterComponent.template).toMatchSnapshot();
    });
});