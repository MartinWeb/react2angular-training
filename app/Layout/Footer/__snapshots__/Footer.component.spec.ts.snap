// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footerComponent should have valid snapshot 1`] = `
"<footer>
    Chuck Norris Â© {{$ctrl.date | date:"d/MM/yyyy"}}
</footer>"
`;
